lib/recog/nizer.rb

```diff
@@ -60,6 +132,48 @@ def self.multi_match(match_key, match_string)
   end
 
   #
+  # Search all fingerprint dbs and attempt to find **all** matching
+  # {Fingerprint fingerprint}s. Returns an array of all matching
+  # fingerprints or an empty array.
+  #
+  # @param match_string [String] Service banner to match
+  # @param [Hash] filters This hash contains filters used to limit the
+  #   results to just those from specific types of fingerprints.
+  #   The matched values are derived from the 'fingerprints' top level
```

Comment:
Minor, but slightly confusing here.  Maybe something like "The value to match against are taken from the 'fingerprints' ..."